platform_events-handler/index.js

```diff
@@ -0,0 +1,119 @@
+const config = require('./components/config.js'); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const async = require("async");
+const AWS = require('aws-sdk');
+const errorHandlerModule = require("./components/error-handler.js");
+const fcodes = require('./utils/failure-codes.js');
+
+
+module.exports.handler = (event, context, cb) => {
+
+	//Initializations
+	var configData = config(context);
+	var errorHandler = errorHandlerModule(logger);
+	const dynamodb = new AWS.DynamoDB();
+	var processedEvents = [];
+	var failedEvents = [];
+
+	var docClient = new AWS.DynamoDB.DocumentClient();
+	var interestedEvents = [];
+	var failureCodes = fcodes();
+	
+	async.series({
```

Comment:
We want to use promises native for all new code to OSS. Can you attempt to convert this to the OSS model code? If you are having to spend more than 1 hour for this, we will revise your estimates.